config/locales/ja.yml

```diff
@@ -0,0 +1,33 @@
+# Files in the config/locales directory are used for internationalization
```

Comment:
Delete this file
@iamdavidpetric what's up doc?

Reply:
done